tests/test_study.py

```diff
@@ -187,8 +179,8 @@ def test_optimize_parallel(n_trials, n_jobs, storage_mode):
 @pytest.mark.parametrize(
     'n_trials, n_jobs, storage_mode',
     itertools.product(
-        (0, 1, 2, MAX_N_TRIALS, None),  # n_trials
-        N_JOBS_LIST,  # n_jobs
+        (0, 1, 10, None),  # n_trials
```

Comment:
IMO, it's better to replace `10` with `20` because `TPESampler` switches the internal algorithm from random search to TPE if the number of finished trials exceeds `10` (the access pattern to the storage radically change between before and after the switching).
```suggestion
        (0, 1, 20, None),  # n_trials
```

Reply:
Thanks, changed as suggested and also for `test_optimize_parallel`. The test time increased by ~10s (from ~77s) which I guess is acceptable.

Reply:
👍